frontend/src/app/modules/home/issues/issues.component.html

```diff
@@ -55,7 +55,9 @@
                     </tr>
                 </ng-template>
                 <ng-template pTemplate="body" let-issue>
-                    <tr>
+                    <tr *ngIf="issue.issueDetails.className !== null &&
```

Comment:
Could you use just issue.issueDetails.className instead of issue.issueDetails.className !== null?
Or there is a reason why you wrote it like this?

Reply:
You are right. I really can simplify the expression here.